### PR TITLE
Keep management commands from crashing unexpectedly in unusual situations.

### PR DESCRIPTION
**Pull request by alvacouch**
Fix for crashes in Jenkins.

### CODE REVIEW

File: hs_core/management/utils.py [hunk 1]
```diff
@@ -182,6 +193,7 @@ def test(self):
             res = BaseResource.objects.get(short_id=self.short_id)
         except BaseResource.DoesNotExist:
             print("{} does not exist in Django".format(self.short_id))
+            return
```
Comment (alvacouch) [inline hs_core/management/utils.py]: This eliminates a lot of erroneous error messages.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by aphelionz** [comment]
@alvacouch should I test and merge these or are we waiting for updates?

**Comment by alvacouch** [comment]
@aphelionz These changes are now complete. They should be run on www in Jenkins.

**Comment by alvacouch** [comment]
@aphelionz The main changes include: 

* get_content_model is run in a try environment and its failure is trapped. 
* Errors are annotated with the resource type and title where appropriate. 
* ingest is run before check, so that error messages don't appear twice.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Pull request merged**